index.bs

```diff
@@ -679,12 +687,17 @@ To <dfn lt="sanitizeFor">sanitize for</dfn> an |element| name of type
 </div>
 
 <div algorithm="sanitizeAndSet">
-To <dfn lt="sanitizeAndSet">sanitize and set</dfn> a |value| using a
-{{Sanitizer}} |sanitizer| on an {{Element}} node |this|, run these steps:
+To <dfn lt="sanitizeAndSet">sanitize and set</dfn> a |value| using an
+{{SetHTMLOptions}} |options| dictionary on an {{Element}} node |this|,
+run these steps:
   1. If the result of running the steps of the
      [=determine the baseline configuration for an element=] algorithm
      for the element |this| is anything other than `keep`, then throw a
      {{TypeError}} and return.
+  1. Let |sanitizer| be [=map/get|the value=] of the {{sanitizer}} member
+     of the |options| {{SetHTMLOptions}} dictionary, or a {{Sanitizer}}
+     instance with the [=default configuration=] if the {{sanitizer}} member
+     does not [=map/exist=].
```

Comment:
Perhaps the following for marginally more precision?

```suggestion
     of the |options| {{SetHTMLOptions}} dictionary, or the result of executing
     [=create a Sanitizer=] with the [=default configuration=] if the {{sanitizer}}
     member does not [=map/exist=].
```

Reply:
Done. I noticed that [=map/get=] isn't actually defined if the key isn't in the map, so I rewrote it a litte more...